SketchUp/sketchup.rb

```diff
@@ -204,7 +202,7 @@ def self.extensions
   # @example
   #   new_sketchup = Sketchup.file_new
   #
-  # @return Sketchup - a Sketchup object if successful.
+  # @return [Sketchup] a Sketchup object if successful.
```

Comment:
NO, it _currently_ actually returns the `Sketchup` **module** object.
(I suppose it could be used in making chained method calls, but this would be unusual for module or class methods.)